ert3/evaluator/_builder.py

```diff
@@ -25,16 +25,19 @@ def add_step_inputs(
     step: StepBuilder,
 ) -> None:
     for input_ in inputs:
+        transformation: ert.data.RecordTransformation = (
```

Comment:
I think since there might be more `RecordTransformations` in the future, we might just add a function `get_recordtransformation(input)` (similarly to get_serializer) to avoid all this ifs.

Reply:
Sounds like a good idea. Will consider if that should be part of this PR or not.

Reply:
Done.